internal/nexodus/nexodus_linux.go

```diff
@@ -92,11 +93,14 @@ func (nx *Nexodus) setupInterfaceOS() error {
 		return fmt.Errorf("%w", interfaceErr)
 	}
 	defer util.IgnoreError(c.Close)
+
+	wgFwMark := 51820
```

Comment:
Can you put this in a constant with some docs in the code that explain what this is for? You can also just reference the design doc to avoid repeating yourself.

Reply:
Yeah, this looks goofy because `FirewallMark *int` and Go, you can't take the address of a constant. Added a const.

```
const 	wgFwMark         = 51820

	fwMark := wgFwMark

	err = c.ConfigureDevice(nx.tunnelIface, wgtypes.Config{
		PrivateKey:   &privateKey,
		ListenPort:   &listenPort,
		ReplacePeers: true,
		Peers:        nil,
		FirewallMark: &fwMark,
	})
```